### PR TITLE
modified connector

### PR DESCRIPTION
**Pull request by cs-sneha-bharote**

### CODE REVIEW

File: aws-dynamodb/info.json [hunk 1]
```diff
@@ -108,7 +108,7 @@
     {
       "title": "Create Table",
       "operation": "create_table",
-      "description": "Adds a new table in Amazon DynamoDB",
+      "description": "Adds a new table in AWS DynamoDB",
       "category": "utilities",
       "enabled": true,
       "parameters": [
```
Comment (cs-pranjali-munfan) [inline aws-dynamodb/info.json:114]: Please add 'Assume a Role' parameter in all the actions

File: aws-dynamodb/operations.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 import functools
 import operator
 
-logger = get_logger('amazon-dynamodb')
+logger = get_logger('aws-dynamodb')
```
Comment (cs-pranjali-munfan) [inline aws-dynamodb/operations.py:13]: update health check for IAM role selection

File: aws-dynamodb/utils.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 import boto3
 from .constant import *
 
-logger = get_logger('amazon-dynamodb')
+logger = get_logger('aws-dynamodb')
```
Comment (cs-pranjali-munfan) [inline aws-dynamodb/utils.py:12]: add logic to generate temporary credentials in case of assume role

**Comment by cs-sneha-bharote** [comment]
Changes have been done by Dilkhush so closing this PR.

**Pull request closed without merging**